src/sparseml/__init__.py

```diff
@@ -53,5 +53,8 @@
     )
 except Exception as err:
     print(
-        f"Need sparsezoo version above 0.9.0 to run Neural Magic's latest-version check\n{err}"
+        f"Need sparsezoo version above 0.9.0 to run Neural Magic's latest-version "
+        f"check\n{err}"
     )
+
+from .pytorch import recipe_template
```

Comment:
we 100% cannot import anything from the pytorch submodule in the main initialization flow. this would do a hard check/fail if torch is installed on every import of `sparseml`. need to remove this

Reply:
Updated